lib/features/local_live_log.js

```diff
@@ -23,27 +23,27 @@ var debug = require('debug')(
 );
 
 // Alias used to link the proxy.
-function TaskclusterLogs() {
-  this.bulkLog = new BulkLog(BACKING_ARTIFACT_NAME);
-}
+export default class TaskclusterLogs {
+  constructor() {
+    this.container = null;
```

Comment:
Makes a lot of sense... It could also be made a member using `container: null`, as `null` is a value type. However, this is a decent pattern, as setting attributes s.t. `constainer: null` only works well for value types, it is a footgun for objects..